modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -1223,6 +1222,72 @@ public void testAddObjectActionWithConditionExpression() throws Exception {
 		_objectActionLocalService.deleteObjectAction(objectAction4);
 	}
 
+	@Test
+	public void testAddObjectActionWithLabelMap() throws Exception {
+		Locale locale = LocaleUtil.fromLanguageId(
+			_objectDefinition.getDefaultLanguageId());
+
+		String objectActionName = RandomTestUtil.randomString();
+
+		ObjectAction objectAction1 = _addObjectAction(
+			StringPool.BLANK, null, objectActionName,
+			ObjectActionExecutorConstants.KEY_GROOVY,
+			ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE,
+			new UnicodeProperties(), false);
+
+		Assert.assertEquals(
+			Collections.singletonMap(locale, objectActionName),
+			objectAction1.getLabelMap());
```

Comment:
@pedro-oliveira446 add comments for each use case, this one can be:

`// label map as null`